### ISSUE TITLE
[Edit] JavaScript: AJAX

### ISSUE THREAD

**Issue by codecademy-docs** [Codecademy/docs#7064]
### Type of Edit (select all that apply)

Add new content (definitions, codeblocks, etc.)

### Description (optional)

We would like to edit the `AJAX` concept entry in JavaScript. The entry lives in `content/javascript/concepts/ajax/ajax.md`.

Please refer to the [concept entry template](https://github.com/Codecademy/docs/blob/main/documentation/concept-entry-template.md), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

Please:

- Add more content
- Add more examples
- Add best practices and/or a FAQ section

### Code of Conduct

- [x] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [x] Labels added
- [ ] Issue is assigned

**Comment by AlbertoJTD** [comment]
Hey @Sriparno08

Are you working on this issue?
I'd like to make a contribution for this topic. I've already my entry with new information about AJAX

**Comment by Sriparno08** [comment]
Yeah @AlbertoJTD, I'm working on this issue.

Please claim another unassigned issue that interests you.

Tip: To ensure your work is taken into consideration, start working on an issue after getting assigned to it.
